package CaseStudyPkg;

public class NestedTryExcep {

	public static void main(String[] args) {
	
		    try
		    {  
			     
		    	int n1 = 10;
				int n2 = 0;
				int n3=n1/n2;
				System.out.println(n3);
			 }
		    catch (ArithmeticException e1) {
				e1.printStackTrace();
		    }
	}
}